### DISCUSSION TITLE
Default preset behavior

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15247] [Feature Requests]
Whenever you create two default presets, for the same `user + collection + layout`, the first one you created is applied, and the second preset is ignored.  Two thoughts:

First, it seems more intuitive for the second *(aka the newest preset)* to take precedence.
Second, it seems like defaults could easily get "lost in the shuffle", and perhaps a warning that another preset exists would be nice.

https://user-images.githubusercontent.com/62306630/185687153-8a6d6e86-5c4a-42d0-bca4-fbc94d3849c3.mov

### To Reproduce

Create two Defaults for  for the same `user + collection + layout`.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
